fastlane/Fastfile

```diff
@@ -6,8 +6,7 @@ lane :setup do
     sh 'git clone https://github.com/artsy/emission.git'
     Dir.chdir('emission') do
       sh 'git checkout skip'
-      sh 'pod repo update'
-      sh 'npm install'
+      sh '. ~/.nvm/nvm.sh && nvm use && npm install yarn --global && yarn install'
```

Comment:
Yeah, but `npm` doesn’t take out lockfile into account, so that’s kind of a no-go. E.g. `npm install` would install the latest React Native, rather than the older version we’re pinned to.